cmd/root.go

```diff
@@ -111,6 +117,18 @@ var RootCmd = &cobra.Command{
 		r.Use(middlewares.CORS())
 		r.Use(middlewares.Logger())
 
+		// Initialize Sentry
+		if sentryDSN != "" {
+			err := sentry.Init(sentry.ClientOptions{
+				Dsn: sentryDSN,
```

Comment:
Can you set `Release` with this [value](https://github.com/ovh/erlenmeyer/blob/master/cmd/version.go#L11) ?
This will be helpfull to know which version are running in Sentry.

Reply:
Done